### PR TITLE
modulesync 2.12.0

### PR DESCRIPTION
**Pull request by bastelfreak**
modulesync 2.12.0

### CODE REVIEW

**Comment by ekohl** [comment]
Restarted the timed out build.

**Comment by bastelfreak** [comment]
I don't quite understand why the Puppet 6 tests are exloding and take over 50 minutes :(

File: .github/workflows/ruby.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+        bundle install --jobs 4 --retry 3
+    - name: execute puppet-lint
+      run: bundle exec rake lint
+    - name: Check syntax of Ruby files and call :syntax and :metadata_lint
+      run: bundle exec rake validate
+    - name: Fails if symlinks are present in directory
+      run: bundle exec rake check:symlinks
+    - name: Fails if .pp files present in tests folder
+      run: bundle exec rake check:test_file
+    - name: Check syntax of Ruby files and call :syntax and :metadata_lint
```
Comment (ekohl) [inline .github/workflows/ruby.yaml:40]: These names appear inconsistent with their actual test

**Comment by bastelfreak** [comment]
I wanted to get proper timings for each rake task. Because of that I explicitly listed every check seperately, instead of using the release_checks. In the future we can probably rely on that again.

File: .github/workflows/ruby.yaml [hunk 2]
```diff
@@ -0,0 +1,47 @@
+name: Ruby
```
Comment (ekohl) [inline .github/workflows/ruby.yaml:1]: I'd be more descriptive here

**Comment by ekohl** [comment]
So I removed the Github actions Puppet 6 change, but now that fails on Travis again because it takes too long.

**Pull request merged**